data/secscan_model/secscan_v4_model.py

```diff
@@ -128,6 +128,8 @@ def load_security_information(self, manifest_or_legacy_image, include_vulnerabil
         try:
             report = self._secscan_api.vulnerability_report(manifest_or_legacy_image.digest)
         except APIRequestFailure as arf:
+            status.delete_instance()
```

Comment:
You'll need to check if we're in read only mode here, and if so, not call this delete (or just catch and ignore the readonlymodeexception)

Reply:
Do we need to add that down where we create the `ManifestSecurityStatus` as well?

Reply:
Nah, since the workers won't be run when in read only mode